packages/storybook/cxl-switch.stories.js

```diff
@@ -0,0 +1,20 @@
+import { withKnobs, text } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-switch.js';
+
+export default {
+  decorators: [withKnobs],
+  title: 'CXL Lumo Styles|Elements'
+};
+
+export const CxlSwitch = () => {
+  const label = text('Label', 'Dark mode');
```

Comment:
Label is too specific for this general toggle, just say "Toggle me", or something.